app/channels/chat_channel.rb

```diff
@@ -1,24 +1,27 @@
+# frozen_string_literal: true
+
 class ChatChannel < ApplicationCable::Channel
   def subscribed
-    if params['room'] == "local_chat"
-      stream_for ChatRoom.local.where(system: current_user.reload.system).first
-    elsif params['room'] == "global_chat"
+    if params["room"] == "local_chat"
```

Comment:
Can we please keep using Option A instead?

`(Option A) Prefer single-quoted strings when you don't need string interpolation or special symbols such as \t, \n, ', etc.`

Reply:
added as a rule in rubocop...